nginx.conf

```diff
@@ -190,7 +190,7 @@ server {
   rewrite ^/academy/web-scraping-for-beginners/crawling/processing-data$ /academy/web-scraping-for-beginners/crawling/exporting-data permanent;
   rewrite ^/academy/web-scraping-for-beginners/crawling/recap-collection-basics$ /academy/web-scraping-for-beginners/crawling/recap-extraction-basics permanent;
   rewrite ^/academy/php/using-apify-scraper-with-php$ /academy/php/use-apify-from-php permanent;
-  rewrite ^/api/?$ /api/v2 redirect;
+  rewrite ^/api/client?$ /api redirect;
```

Comment:
i think should be `/?`, meaning we redirect on `/api/client` and `/api/client/` but not `/api/client/xxx`. the way you have it means the `t` is optional, so its `/api/client` or `/api/clien`